tools/sim/start_carla.sh

```diff
@@ -1,18 +1,20 @@
-#!/bin/bash -e
+#!/bin/bash
 
-FILE=CARLA_0.9.7.tar.gz
-if [ ! -f $FILE ]; then
-  curl -O http://carla-assets-internal.s3.amazonaws.com/Releases/Linux/$FILE
-fi
-if [ ! -d carla ]; then
-  rm -rf carla_tmp
-  mkdir -p carla_tmp
-  cd carla_tmp
-  tar xvf ../$FILE
-  easy_install PythonAPI/carla/dist/carla-0.9.7-py3.5-linux-x86_64.egg || true
-  cd ../
-  mv carla_tmp carla
-fi
+# Requires nvidia docker - https://github.com/NVIDIA/nvidia-docker
```

Comment:
Find a good check if `nvidia-docker` is installed. Maybe something with `apt`?

Reply:
It’s commented out below this line

Reply:
yeah, the `which nvidia-docker` check doesn't work, that's why it's commented out.